docs/tasks/configure-pod-container/downward-api-volume-expose-pod-information.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Exposing Pod Information Using a DownwardApiVolumeFile
+---
+
+{% capture overview %}
+
+This page shows how a Pod can use a DownwardAPIVolume to expose information
+about itself to Containers running in the Pod. A DownwardAPIVolume can expose
+Pod fields and Container fields.
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Storing Pod fields
+
+In this exercise, you create a Pod that has one Container.
+Here is the configuration file for the Pod:
+
+{% include code.html language="yaml" file="dapi-volume.yaml" ghlink="/docs/tasks/configure-pod-container/dapi-volume.yaml" %}
```

Comment:
The yaml line "command: ["sh", "-c", "while true; do if [[ -e /etc/labels ]]; then cat /etc/labels; fi; if [[ -e /etc/annotations ]]; then echo -en '\n\n'; cat /etc/annotations; fi; sleep 5; done"]" is really long and goes off screen in the preview for me, and there's no way for me to scroll to see the rest of it. Is this only a problem w/ the Netlify preview, or will that also be the case when it's published? If the latter, is there a way to make it wrap or scroll?